.git-commit-template.txt

```diff
@@ -0,0 +1,31 @@
+# HEADER
+# |<----  Using a Maximum Of 50 Characters  ---->|
+<type>(<scope>): <subject>
+# Type can be 
+#    feat     (A new feature)
+#    fix      (A bug fix)
+#    docs     (Documentation only changes)
+#    style    (Changes that do not affect the meaning of the code e.g. white-space, formatting, missing semi-colons, etc)
+#    refactor (A code change that neither fixes a bug nor adds a feature)
+#    perf     (A code change that improves performance)
+#    test     (Adding missing or correcting existing tests)
+#    chore    (Changes to the build process or auxiliary tools and libraries such as documentation generation)# Type must cope
```

Comment:
Shouldn't there be a new line before `# Type`?

Reply:
Looks like the part after the # just shouldn't be there at all, not sure how it got there tbh. Thanks for noticing.